applications/DEMApplication/tests/test_DEM_2D_restitution.py

```diff
@@ -45,7 +38,9 @@ def Finalize(self):
             Logger.PrintInfo("upper bound:", restitution_coefficient*tolerance)
             Logger.PrintInfo("lower bound:", restitution_coefficient/tolerance)
             self.assertAlmostEqual(self.coeff, restitution_coefficient, delta=tolerance)
+            check_mark_1 = True
 
+        self.assertTrue(check_mark_1)
```

Comment:
I think this is a debugging leftover.

Reply:
Actually, it is not. Explanation: since these DEM tests were written, there have been several occasions when the time step was modified, or the total time was modified, because we needed to make the tests shorter. When you do these changes, there is a high risk that you do not go through all the asserts. WIth this 'manual' trick I make sure that all asserts have been called. I looked for a buil-in tool in Unittcase (which makes sure that all asserts have been called) but I did not find it.